src/common/types.ts

```diff
@@ -333,3 +333,38 @@ export class WebSocketSession extends EventEmitter {
    */
   sessionId?: string
 }
+
+/**
+ * Basic information of equipment
+ */
+export interface EquipmentInfo {
```

Comment:
Check the optional fields. Now these types indicate that all the fields would be always present.